chia/_tests/core/test_full_node_rpc.py

```diff
@@ -88,8 +88,12 @@ async def test1(
 
         peak_block = await client.get_block(state["peak"].header_hash)
         assert peak_block == blocks[-1]
-        assert (await client.get_block(bytes32([1] * 32))) is None
-
+        try:
+            await client.get_block(bytes32([1] * 32))
+        except ValueError as e:
+            assert str(e).endswith("not found")
+        else:
+            assert False, "Expected ValueError for non-existent block"
```

Comment:
```suggestion
        with pytest.raises(ValueError, matches="not found"):
            await client.get_block(bytes32([1] * 32))
```
I think every test in your patch would benefit from this change

Reply:
good suggestion